frontend/src/hooks/useUiFlag.ts

```diff
@@ -2,8 +2,14 @@ import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
 
 type flags = ReturnType<typeof useUiConfig>['uiConfig']['flags'];
 
-export const useUiFlag = <K extends keyof flags>(flag: K) => {
+export const useUiFlag = <K extends keyof flags>(flag: K): boolean => {
     const { uiConfig } = useUiConfig();
+    const value = uiConfig?.flags?.[flag];
+    if (typeof value === 'boolean') {
+        return value;
+    } else if (typeof value !== 'undefined') {
+        console.error(`Flag ${flag} does not return a boolean.`);
+    }
 
-    return uiConfig?.flags?.[flag] || false;
+    return false;
```

Comment:
While I think this is fine in theory, I'm not confident about changing it now because we have places where we depend on the contents. Yeh, ts found the one in feedback component, but I don't really trust it to monitor everywhere. I don't see what we gain by changing this function right now. I'd rather look at other options too before changing this one.